### PR TITLE
feat(contact): added contact page with form and mailer system

### PR DESCRIPTION
**Pull request by WizBhoo**
Hello Yann,

Could you please review this feature ?
I added the contact page with the appropriate form and i have implemented a mailer system using google-mailer.

Thanks.

### CODE REVIEW

File: src/Mailer/MailerMessage.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+     * @param Contact $contact
+     *
+     * @return void
+     *
+     * @throws TransportExceptionInterface
+     */
+    public function sendContactEmail(Contact $contact): void
+    {
+        $email = (new TemplatedEmail())
+            ->from(new Address('wizbhoo.dev@gmail.com', 'APi - Contact Form'))
```
Comment (drixs6o9) [inline src/Mailer/MailerMessage.php:49]: Use the real sender email no ?

File: templates/emails/contactHTML.html.twig [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+<ul>
+    <li>Nom : {{ contact.firstname }} {{ contact.lastname }}</li>
+    <li>Email : {{ contact.email }}</li>
+    <li>Sujet : {{ contact.subject }}</li>
+    <li>
+        Message :<br>
+        {{ contact.message | nl2br }}
+    </li>
+</ul>
```
Comment (drixs6o9) [inline templates/emails/contactHTML.html.twig:11]: Email template must have html / body tags !

File: src/Mailer/MailerMessage.php (hunk 1 above)
Comment (WizBhoo) [reply src/Mailer/MailerMessage.php:49]: It's this one : the mail is sent from wizbhoo.dev

File: src/Mailer/Sender.php [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+     * @param TemplatedEmail $email
+     *
+     * @return void
+     *
+     * @throws TransportExceptionInterface
+     */
+    public function sendContactEmail(TemplatedEmail $email): void
+    {
+        $this->mailer->send($email);
+    }
```
Comment (drixs6o9) [inline src/Mailer/Sender.php:47]:
```suggestion
    /**
     * Send email.
     *
     * @param TemplatedEmail $email
     *
     * @return void
     *
     * @throws TransportExceptionInterface
     */
    public function sendEmail(TemplatedEmail $email): void
    {
        $this->mailer->send($email);
    }
```

File: src/Manager/ContactManager.php [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    /**
+     * Build a contact email.
+     *
+     * @param array $contact
+     *
+     * @return void
+     *
+     * @throws TransportExceptionInterface
+     */
+    public function buildContactEmail(array $contact): void
```
Comment (drixs6o9) [inline src/Manager/ContactManager.php:45]:
```suggestion
    public function sendContactEmail(array $contact): void
```

File: src/Manager/ContactManager.php [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+            ->from(new Address('wizbhoo.dev@gmail.com', 'APi - Contact Form'))
+            ->to(new Address('apierrard.contact@gmail.com', 'APi - CV & Portfolio'))
+            ->replyTo($contact['email'])
+            ->subject($contact['subject'])
+            ->htmlTemplate('emails/contactHTML.html.twig')
+            ->context(
+                [
+                    'contact' => $contact,
+                ]
+            )
```
Comment (drixs6o9) [inline src/Manager/ContactManager.php:57]:
```suggestion
            ->context([
                'contact' => $contact,
            ])
```

**Review by drixs6o9 — CHANGES_REQUESTED** [review]
No tests ?

**Review by drixs6o9 — APPROVED** [review]

**Pull request merged**